ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsTransactionCartList.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  HSpacer,
+  IOStyles,
+  ListItemTransaction,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { CartItem } from "../../../../../definitions/pagopa/biz-events/CartItem";
+import { formatAmountText } from "../utils";
+
+type Props = {
+  carts?: ReadonlyArray<CartItem>;
+  loading: boolean;
+  onPress: (cartItem: CartItem) => void;
+};
+
+/**
+ * This component renders a list of transaction cart details
+ */
+export const PaymentsBizEventsTransactionCartList = ({
+  carts,
+  loading,
+  onPress
+}: Props) => {
+  if (loading) {
+    return <SkeletonTransactionDetailsList />;
+  }
+  if (!carts) {
+    return <></>;
```

Comment:
```suggestion
    return null;
```
Prefer using null we you need to return "empty components".
`null` is nothing, an empty `React.Fragment` is still something so it's slower.

Reply:
Addressed into 90bdac3da7e0ddc57949cbdd54da09aa5dc0d231